src/planeta/utils.js

```diff
@@ -35,12 +35,22 @@ const LEAP_COLOR = 0;
 const CO2_COLOR = 2;
 const GOELLARS_COLOR = 3;
 
+export const choice = l => l[Math.floor(Math.random() * l.length)];
```

Comment:
'l' is kinda hard to read here. Can you change to to something like `arr` at least? :)